src/cmd-buildextend-installer

```diff
@@ -34,6 +35,10 @@ if not args.build:
 
 print(f"Targeting build: {args.build}")
 
+with open('src/config/image.yaml') as fh:
+    image_yaml = yaml.safe_load(fh)
+squashfs_compression = image_yaml.get('squashfs-compression', 'zstd')
```

Comment:
I think we need this in `IMAGE_CONF_KEYS` in `virt-install` too...which appears to be the only thing validating the "schema" right now, need to move that into `cmd-buildextend-metal`.